inc/internal/az_ulib_ipc.h

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#ifndef INTERNAL_AZ_ULIB_IPC_H
+#define INTERNAL_AZ_ULIB_IPC_H
+
+#include "azure_macro_utils/macro_utils.h"
+#include "umock_c/umock_c_prod.h"
+
+#include "../az_ulib_action_api.h"
+#include "../az_ulib_descriptor_api.h"
+#include "az_pal_os_api.h"
+#include "az_ulib_action_api.h"
+#include "az_ulib_base.h"
+#include "ulib_config.h"
+#include "ulib_port.h"
+#include "ulib_result.h"
+
+#ifndef __cplusplus
+#include <stdbool.h>
```

Comment:
is this needed?

Reply:
No, removed.

---
In reply to: [338294442](https://github.com/Azure/azure-ulib-c/pull/40#discussion_r338294442) [](ancestors = 338294442)